From 30afaf17b2a567daad6efd467b0d6a0a10b3d527 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Wed, 17 Jul 2013 15:22:20 +0200
Subject: [PATCH 0821/1566] GPIO: xilinx: Use standard coding style for macros

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Trivial.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 6ecc02b189fb0147dd3221ff347f42901c575927)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpio/gpio-xilinx.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpio/gpio-xilinx.c b/drivers/gpio/gpio-xilinx.c
index d29adfb..f2d073a 100644
--- a/drivers/gpio/gpio-xilinx.c
+++ b/drivers/gpio/gpio-xilinx.c
@@ -29,8 +29,8 @@
 #include <linux/slab.h>
 
 /* Register Offset Definitions */
-#define XGPIO_DATA_OFFSET   (0x0)	/* Data register  */
-#define XGPIO_TRI_OFFSET    (0x4)	/* I/O direction register  */
+#define XGPIO_DATA_OFFSET	0x0 /* Data register */
+#define XGPIO_TRI_OFFSET	0x4 /* I/O direction register */
 #define XGPIO_GIER_OFFSET	0x11c /* Global Interrupt Enable */
 #define XGPIO_GIER_IE		BIT(31)
 
-- 
1.7.5.4

